btrfs.go

```diff
@@ -328,11 +328,11 @@ func SubvolDelete(path string) error {
 		}
 
 		if err := isFileInfoSubvol(fi); err != nil {
-			return err
+			return nil
```

Comment:
If you're ignoring `err`, what's the point of calling `isFileInfoSubvol` in the first place?